src/gui/sdlmain.cpp

```diff
@@ -584,6 +628,15 @@ static SDL_Window * GFX_SetSDLWindowMode(Bit16u width, Bit16u height, bool fulls
 		const int sdl_pos = SDL_WINDOWPOS_UNDEFINED_DISPLAY(sdl.display_number);
 		sdl.window = SDL_CreateWindow("", sdl_pos, sdl_pos, width, height, flags);
 
+#if SDL_VERSION_ATLEAST(2, 0, 5)
+		if (resizable) {
+			SDL_AddEventWatch(watch_sdl_events, sdl.window);
+			SDL_SetWindowResizable(sdl.window, SDL_TRUE);
+			sdl.desktop.window.resizable = true;
+		} else {
+			sdl.desktop.window.resizable = false;
+		}
+#endif
```

Comment:
Can the assignment of `sdl.desktop.window.resizable` be hoisted out of `SetWindowMode` and up to the level where `resizable` is determined? (because it doesn't look like `SetWindowMode` makes any new decisions regarding resizeability)

Reply:
Moving this outside of `SetWindowMode` would move it into ~6 locations in the code… And I find it a bit reassuring, that this flag is being updated after the new window is created (that's the reason why we have 2 separate flags: `sdl.desktop.want_resizable_window` and `sdl.desktop.window.resizable` - when there was 1 flag I was facing awful corner-cases to handle when switching between Mapper window and emulator window).

That being said, this code should definitely switch places with nullcheck below, and there's no need to update flags inside if/else any more.

I will update and simplify this area.

Reply:
> Moving this outside of SetWindowMode would move it into ~6 locations in the code

Thanks for explaining, and yeah, that would be much worse.  Also agree with all your other points.